src/Main.elm

```diff
@@ -231,8 +227,8 @@ viewBody model =
         [ text <| "Authenticating, please wait..." ]
 
 
-formView : Form () Proposal -> Html Form.Msg
-formView form =
+formView : Model -> Html Msg
+formView model =
   let
     errorFor field =
```

Comment:
These definitions are not used in the function's body. Whole let clause can be removed.